clarity/src/vm/types/mod.rs

```diff
@@ -233,6 +239,7 @@ pub enum Value {
     Tuple(TupleData),
     Optional(OptionalData),
     Response(ResponseData),
+    Trait(TraitData),
```

Comment:
I think `Trait` might be a confusing name here -- this Value isn't really a trait, it's a dynamically-callable contract, right? so maybe something like `CallableContract` would be better?

Reply:
I like it.

Reply:
Changed in 9f16195e0